docs/config-file-package-configuration-yml.md

```diff
@@ -51,24 +56,69 @@ curations have the same syntax and semantic as the analog entries for the `ort.y
     concluded_license: "Apache-2.0"
 ```
 
-## Organizing package configurations
+## Command Line
 
-ORT currently provides two alternatives for organizing package configurations to choose from. It is required for both
-options that there is at most one package configuration provided for any (package, provenance) combination.
+ORT offers two different ways to use package configurations:
 
-### Configuration directory
+- A single configuration `.yml` containing an array with each entry defining the configuration for one package
+- A directory with configuration files for each `package id`, with one file for each configured package/provenance combination
 
-A directory containing one `.yml` file for each configured (package, provenance) combination.
-Each `.yml` file contains exactly one package configuration as shown in the parent sections.
-This directory can be specified via the `--package-configuration-dir` option to the *evaluator* and the *reporter*
-command respectively.  
- 
-### Configuration file
+Note that in both of the above options only one package configuration can exist for a package/provenance combination.
+
+### Using a Package Configuration Directory
 
-A single `.yml` containing one large array with each entry being one package configuration as shown in the parent
-section. This file can be specified via the `--package-configuration-file` option to the *evaluator* and the *reporter*
-command respectively. For example such single file would look as follows:
+To use a directory with configuration files for each `package id`, pass it to the `--package-configuration-dir` option of the _evaluator_:
+
+```bash
+cli/build/install/ort/bin/ort evaluate
+  -i [scanner-output-path]/scan-result.yml
+  -o [evaluator-output-path]/evaluator-result.yml
+  --output-formats YAML
+  --license-configuration-file [ort-configuration-path]/licenses.yml
+  --package-curations-file [ort-configuration-path]/curations.yml
+  --package-configuration-dir [ort-configuration-path]/packages
```

Comment:
is hit possible to highligt this e.g. via red color?

Reply:
No, we are using (GitHub flavored) Markdown for our docs and Markdown is very simple so no highlighting within codeblocks